src/applications/personalization/profile/mocks/server.js

```diff
@@ -4,7 +4,11 @@ const paymentHistory = require('./paymentHistory');
 const mhvAcccount = require('./mhvAccount');
 const address = require('./address');
 const status = require('./status');
-const personalInformation = require('./personalInformation');
+const {
+  handlePutGenderIdentitiesRoute,
+  handleGetPersonalInformationRoute,
+  handlePutPreferredNameRoute,
+} = require('./personal-information');
```

Comment:
Moved this mock into its own folder to adhere to the other mock's organizational structure